### PR TITLE
Update testing

### PR DESCRIPTION
**Pull request by Antony1060**

### CODE REVIEW

**Comment by Antony1060** [comment]
Someone please change the README to have docs for the non tree-shakeable functions and push to this branch.

File: src/index.ts [hunk 1]
```diff
@@ -48,17 +48,21 @@ export const fromPermissionsBuffer = (buffer: Buffer) => {
     return bits;
 };
 
-export const toPermissionsBitString = (data: PermissionData) => data.toString(2);
+export const toPermissionsBitString = (data: PermissionData) =>
+    data.toString(2);
 
-export const generatePermissions = (root: bigint) => {
+export const newPermissions = (root: bigint = EMPTY_PERMISSIONS) => {
```
Comment (lucemans) [inline src/index.ts]: Not sure about the use of the reserved keyword `new`..
Default is a smart move!

**Review by lucemans — CHANGES_REQUESTED** [review]
Not sure about the `new` keyword, what do you think about `createPermissions` `instantiatePermissions` `makePermissions`.

I think I like create better, but not sure

File: src/index.ts (hunk 1 above)
Comment (Antony1060) [reply src/index.ts]: I like `make` and `create`. So let's go with `create`.

**Review by svemat01 — APPROVED** [review]
LGTM

**Review by lucemans — APPROVED** [review]
LGTM

**Pull request merged**